params/network_upgrades.go

```diff
@@ -4,14 +4,18 @@
 package params
 
 import (
+	"time"
+
 	"github.com/ava-labs/avalanchego/utils/constants"
 	"github.com/ava-labs/subnet-evm/utils"
 )
 
 var (
+	defaultActivationTime = time.Date(2020, time.December, 5, 5, 0, 0, 0, time.UTC)
+
 	LocalNetworkUpgrades = MandatoryNetworkUpgrades{
 		SubnetEVMTimestamp: utils.NewUint64(0),
-		DUpgradeTimestamp:  utils.NewUint64(0),
+		DUpgradeTimestamp:  utils.TimeToNewUint64(defaultActivationTime),
```

Comment:
Giving a zero timestamp here means upgrade will be activated at genesis. I don't know if we actually want this or we want to activate those after the first block. Reverted this change.

Reply:
Imo for the local network, we should activate the `DUpgrade` on local network upgrades at genesis when we activate on fuji/mainnet the same way we handle this for activating network upgrades on the local network on coreth rather than a non-zero default activation time.